build/MSBuildToolset/project.json

```diff
@@ -12,7 +12,7 @@
     "Microsoft.Build.Targets": "0.1.0-preview-00023-160527",
     "Microsoft.Build": "0.1.0-preview-00023-160527",
     "MSBuild": "0.1.0-preview-00023-160527",
-    "Microsoft.NETCore.TestHost": "1.0.0-rc3-24206-00",
+    "Microsoft.NETCore.TestHost": "1.0.0-rc3-24208-01",
```

Comment:
I think this line should be -00
https://github.com/dotnet/versions/blob/1b3784bee6ec0e71f17b889f8a2c507701fd6398/build-info/dotnet/projectk-tfs/release/1.0.0/Latest_Packages.txt#L4